src/shared/components/palette/dynamic-list/write-dynamic-list-field.component.ts

```diff
@@ -20,15 +20,15 @@ export class WriteDynamicListFieldComponent extends AbstractFieldWriteComponent
     }
 
     /**
-     *
      * Reassigning value from formatted_value when value is empty
      */
-    if (!this.caseField.value && this.caseField.formatted_value && this.caseField.formatted_value.value && this.caseField.formatted_value.value.code) {
-      this.caseField.value = this.caseField.formatted_value.value.code;
+    if (!this.caseField.value) {
```

Comment:
Is it breaking if this remains in it's previous form, as in one big `if` with the other conditions?

Reply:
Yes, it's a lint error because the line is more than 140 characters long.